### ISSUE TITLE
NativeWind V4, Expo-router: Tailwind ClassName styles not being automatically applied at auto-refresh/hot-reload

### ISSUE THREAD

**Issue by onyedikachi23** [nativewind/nativewind#1182]
**Describe the bug**
Whenever I add new styles to the `className` prop, saving the file won't automatically apply the styles to the component. 

I have to either restart the server, clear bundler cache, or refresh many times, which is a nightmare.

**Reproduction**

Just setup the nativewind V4 using the guide for `expo-router` and write `className` styles and see if it works immediately on hot reload of the app.

**Expected behavior**

styles added to the className prop should be applied immediately on saving file - hot reload.

**Additional context**
Nothing

**Comment by marklawlor** [comment]
Please add a reproduction repro. These issues tend to be cause by the projects configuration so I cannot provide you with an answer unless I'm able to run the code.

**Comment by onyedikachi23** [comment]
Here's the reproduction repo - [https://github.com/onyedikachi23/nativewind-issue-reproduction-repo](https://github.com/onyedikachi23/nativewind-issue-reproduction-repo)

**Comment by marklawlor** [comment]
I clone your project and fast-refresh is working without any modifications.

This suggests the problem is something environmental to your project / Metro. I would recommend clearing the Metro cache.

* What happens if you run the TailwindCLI in watch mode? `npx tailwindcss -i ./global.css --watch`, does it update when you make changes?
* Trying running with `DEBUG=nativewind`. This outputs extra information in the terminal that may help you identify a problem
* Try creating a new project from a different template like https://rn.new/ . This is more to isolate if there is a problem with just this single project, or all projects

**Comment by bojandurmic** [comment]
Hey @marklawlor , this bug happens to me and I have an interesting theming system. This is my `global.css` file:

```
@tailwind base;
@tailwind components;
@tailwind utilities;

@layer base {
  :root {
    --text-primary: 28 28 30;
    --text-secondary: 99 99 102;
    --text-tertiary: 142 142 147;
    --text-quaternary: 174 174 178;
  }

  @media (prefers-color-scheme: dark) {
    :root {
      --text-primary: 242 242 247;
      --text-secondary: 174 174 178;
      --text-tertiary: 142 142 147;
      --text-quaternary: 99 99 102;
    }
  }
}
```

And then in my Tailwind config, under `extend`, I do:

```
colors: {
  text: {
    primary: "rgb(var(--text-primary) / <alpha-value>)",
    secondary: "rgb(var(--text-secondary) / <alpha-value>)",
    tertiary: "rgb(var(--text-tertiary) / <alpha-value>)",
    quaternary: "rgb(var(--text-quaternary) / <alpha-value>)",
  }
}
```

If I add a class `text-text-primary`, it won't work until I reload the app. Then, it will work fine. But then again, if I change the class to `text-text-secondary`, it will fail until I refresh.

This all works great in regular Tailwind CSS on the web.

**Comment by onyedikachi23** [comment]
@marklawlor, what @bojandurmic says is exactly same as mine.

**TailwindCSS** on the web works very fine.

I'm also sorry I haven't done the tasks you said I should do. The bug made me abandon the project for now and move to something else.

**Comment by bojandurmic** [comment]
@onyedikachi23 Expo 52 beta, Mac OS, iOS real device, using custom development client. But I also bring some good news for you, sir, as I have managed to resolve the issue, and this is how:

1. Create a custom component, I called it `ThemeContainer`:

```
import { vars } from "nativewind";
import { View } from "react-native";
import { FC, ReactNode } from "react";

import useColorScheme from "@/hooks/useColorScheme";

const theme = {
  light: {
    "--text-primary": "28 28 30",
    "--text-secondary": "99 99 102",
    "--text-tertiary": "142 142 147",
    "--text-quaternary": "174 174 178",
  },
  dark: {
    "--text-primary": "242 242 247",
    "--text-secondary": "174 174 178",
    "--text-tertiary": "142 142 147",
    "--text-quaternary": "99 99 102",
  },
} as const;

const ThemeContainer: FC<{ children?: ReactNode }> = ({ children }) => {
  const colorScheme = useColorScheme();
  return (
    <View className="flex-1" style={vars(theme[colorScheme])}>
      {children}
    </View>
  );
};

export default ThemeContainer;

```

I have a custom `useColorScheme`, but this is all it does:

```
import { useColorScheme as useColorSchemeRN } from "react-native";

const useColorScheme = (): "light" | "dark" => {
  const colorScheme = useColorSchemeRN();
  return colorScheme || "light";
};

export default useColorScheme;

```

2. Put this `ThemeContainer` component at your root level. Since I am using Expo Router, I put it in `app/_layout.tsx`.

```
export default function RootLayout() {
  verifyInstallation();
  const colorScheme = useColorScheme();
  const [loaded] = useFonts({
    SpaceMono: require("../assets/fonts/SpaceMono-Regular.ttf"),
  });

  useEffect(() => {
    if (loaded) {
      SplashScreen.hideAsync();
    }
  }, [loaded]);

  if (!loaded) {
    return null;
  }

  return (
    <ThemeProvider value={colorScheme === "dark" ? DarkTheme : DefaultTheme}>
      <ThemeContainer>
        <GestureHandlerRootView className="flex-1">
          <StatusBar style="auto" />
          <QueryClientProvider client={queryClient}>
            <AuthProvider>
              <Slot />
            </AuthProvider>
          </QueryClientProvider>
        </GestureHandlerRootView>
      </ThemeContainer>
    </ThemeProvider>
  );
}
```

And now it all works fine, no longer requiring me to reload the entire app.

**Comment by marklawlor** [comment]
Can someone please answer my first question whether you still see this issue when only running Tailwind CLI.

NativeWind doesn't process file changes or setup any file listeners. It starts Tailwind CLI in watch mode, and when it produces new styles from the outputted stylesheet. So if TailwindCLI is not working correctly, then NativeWind won't.

Once isolated to NativeWind issue, please provide full reproduction repros. Issues like this can have many factors and when users provide file snippets, they rarely include the problem, which leads to me being unable to reproduce the issue

**Comment by bojandurmic** [comment]
@marklawlor when I run Tailwind in watch mode, I see the styles I've used being correctly output in `output.css` file.

This has to be a NativeWind issue, because moving my variable declarations from `global.css` file to a React Native component with `vars` made it all work properly.

Plus, it's not like NativeWind just doesn't see those styles. It applies them correctly, but only after I hit a reload button in my Terminal.

Let me see if I can make a small reproduction repository this weekend.

**Comment by j0j032** [comment]
I got the exact same error set up new expo 52 project yesterday. 
My global.css variables for colors work only after an explicit reload. 
The tailwind output is ok.

**Comment by mehmetpolat** [comment]
I experienced the same issue and realized that the problem is caused by using the global.css file within `app/_layout.tsx`

Here's how I solved it:

I created a `NativewindProvider`, imported the global.css file there, and used it within `app/_layout.tsx`

```typescript
// NativewindProvider.tsx

import "../global.css";

export default function NativewindProvider({
  children,
}: {
  children: React.ReactNode;
}) {
  return <>{children}</>;
}
```

```typescript
// app/_layout.tsx

function RootLayoutNav() { 
  return ( 
      <NativewindProvider>
        <Slot />
      </NativewindProvider> 
  );
}
```

**Comment by marklawlor** [comment]
To the new people replying to the issue. Simply saying "I have this issue" or "I have this work around" is *not* productive in getting the original issue fixed.

I'd like to resolve the original issue, which requires people to provide more information. I'm marking your comments as off-topic, because you are just adding noise to this issue. If you would like a fix, please assist me in providing a minimal reproduction repro where you can reliable reproduce this issue.

@mehmetpolat That workaround can cause issues depending on `import` order in your `_layout` file.

**Comment by mehmetpolat** [comment]
@marklawlor  okey, I deleted my comment. 

Repo: https://github.com/mehmetpolat/hot-reload-problem

Video: https://jumpshare.com/s/gPnhzxmlNg3sxCrg8mqa

**Comment by marklawlor** [comment]
@mehmetpolat The reproduction repro does not have NativeWind installed. Is this missing some commits?

**Comment by mehmetpolat** [comment]
> @mehmetpolat The reproduction repro does not have NativeWind installed. Is this missing some commits?

sorry. Updated

**Comment by marklawlor** [comment]
@mehmetpolat From your repo and your video it appears that you have a different issue. The styles are being updated on Fast Refresh, but your issue is that its changing you from  Tab 2 -> Tab 1.

That seems to be a very different issue that what is being reported here

Can you confirm, or am i understanding your issue incorrectly?

**Comment by mehmetpolat** [comment]
> @mehmetpolat From your repo and your video it appears that you have a different issue. The styles are being updated on Fast Refresh, but your issue is that its changing you from  Tab 2 -> Tab 1.
> 
> 
> 
> That seems to be a very different issue that what is being reported here
> 
> 
> 
> Can you confirm, or am i understanding your issue incorrectly?

Yes, I misunderstood and messed with this topic and now I realized when I read it again. My problem is that it goes back to the first route during hot reload. If you want I can delete the comments and create a new topic without causing more confusion here.

**Comment by mehmetpolat** [comment]
> > @mehmetpolat From your repo and your video it appears that you have a different issue. The styles are being updated on Fast Refresh, but your issue is that its changing you from  Tab 2 -> Tab 1.
> > That seems to be a very different issue that what is being reported here
> > Can you confirm, or am i understanding your issue incorrectly?
> 
> Yes, I misunderstood and messed with this topic and now I realized when I read it again. My problem is that it goes back to the first route during hot reload.I deleted all comments. I will open a new issue with details

**Comment by j0j032** [comment]
Hello Mark, sorry for the last comment without much context...

So, [here's my reproduction repo](https://github.com/j0j032/repro-nativewind)

Init an expo 52 beta project, setup nativewind, created some custom colors in global.css and setup the tw config...

If I use the color for the first time, the color is not applied until I explicitly reload the app. Then, If I go back to an already used color, the style apply correctly. 

https://github.com/user-attachments/assets/6afc8e88-46ac-472b-93c1-ab8e6386594e
Expo 52 beta, Mac OS, iOS 18.1 & Android simulator -> Expo Go and dev build 

Thanks

**Comment by kiikoh** [comment]
I had a similar issue. I was able to solve by deleting a ton of cached files. I always start my project with `expo start -c` so that alone is not enough.

**Comment by spylefkaditis** [comment]
This might be a dumb question. But have you guys updated your tailwind.config files to scan all relevant folders? If your components live inside of a **/src** directory for example, they wont be picked up if you're simply following the setup guide as it only includes the **/app** directory

**Comment by sbkl** [comment]
As a workaround, just adding all possible utilities involving the custom vars in the css files in a utils file made it work for me, even with hsl vars. No need to make a theme container. Just by having the actual utility present inside the project at build time makes the css vars available.

```typescript
export const textColors = [
  "text-background",
  "text-foreground",
  "text-card",
  "text-card-foreground",
  "text-popover",
  "text-popover-foreground",
  "text-primary",
  "text-primary-foreground",
  "text-secondary",
  "text-secondary-foreground",
  "text-muted",
  "text-muted-foreground",
  "text-accent",
  "text-accent-foreground",
  "text-destructive",
  "text-destructive-foreground",
  "text-border",
  "text-input",
  "text-ring",
  "text-sidebar-background",
  "text-sidebar-foreground",
  "text-sidebar-primary",
  "text-sidebar-primary-foreground",
  "text-sidebar-accent",
  "text-sidebar-accent-foreground",
  "text-sidebar-border",
  "text-sidebar-ring",
];

export const backgroundColors = [
  "bg-background",
  "bg-foreground",
  "bg-card",
  "bg-card-foreground",
  "bg-popover",
  "bg-popover-foreground",
  "bg-primary",
  "bg-primary-foreground",
  "bg-secondary",
  "bg-secondary-foreground",
  "bg-muted",
  "bg-muted-foreground",
  "bg-accent",
  "bg-accent-foreground",
  "bg-destructive",
  "bg-destructive-foreground",
  "bg-border",
  "bg-input",
  "bg-ring",
  "bg-sidebar-background",
  "bg-sidebar-foreground",
  "bg-sidebar-primary",
  "bg-sidebar-primary-foreground",
  "bg-sidebar-accent",
  "bg-sidebar-accent-foreground",
  "bg-sidebar-border",
  "bg-sidebar-ring",
];

export const borderColors = [
  "border-background",
  "border-foreground",
  "border-card",
  "border-card-foreground",
  "border-popover",
  "border-popover-foreground",
  "border-primary",
  "border-primary-foreground",
  "border-secondary",
  "border-secondary-foreground",
  "border-muted",
  "border-muted-foreground",
  "border-accent",
  "border-accent-foreground",
  "border-destructive",
  "border-destructive-foreground",
  "border-border",
  "border-input",
  "border-ring",
  "border-sidebar-background",
  "border-sidebar-foreground",
  "border-sidebar-primary",
  "border-sidebar-primary-foreground",
  "border-sidebar-accent",
  "border-sidebar-accent-foreground",
  "border-sidebar-border",
  "border-sidebar-ring",
];

export const divideColors = [
  "divide-background",
  "divide-foreground",
  "divide-card",
  "divide-card-foreground",
  "divide-popover",
  "divide-popover-foreground",
  "divide-primary",
  "divide-primary-foreground",
  "divide-secondary",
  "divide-secondary-foreground",
  "divide-muted",
  "divide-muted-foreground",
  "divide-accent",
  "divide-accent-foreground",
  "divide-destructive",
  "divide-destructive-foreground",
  "divide-border",
  "divide-input",
  "divide-ring",
  "divide-sidebar-background",
  "divide-sidebar-foreground",
  "divide-sidebar-primary",
  "divide-sidebar-primary-foreground",
  "divide-sidebar-accent",
  "divide-sidebar-accent-foreground",
  "divide-sidebar-border",
  "divide-sidebar-ring",
];

export const shadowColors = [
  "shadow-background",
  "shadow-foreground",
  "shadow-card",
  "shadow-card-foreground",
  "shadow-popover",
  "shadow-popover-foreground",
  "shadow-primary",
  "shadow-primary-foreground",
  "shadow-secondary",
  "shadow-secondary-foreground",
  "shadow-muted",
  "shadow-muted-foreground",
  "shadow-accent",
  "shadow-accent-foreground",
  "shadow-destructive",
  "shadow-destructive-foreground",
  "shadow-border",
  "shadow-input",
  "shadow-ring",
  "shadow-sidebar-background",
  "shadow-sidebar-foreground",
  "shadow-sidebar-primary",
  "shadow-sidebar-primary-foreground",
  "shadow-sidebar-accent",
  "shadow-sidebar-accent-foreground",
  "shadow-sidebar-border",
  "shadow-sidebar-ring",
];

export const radiusCorners = [
  "rounded-t-lg",
  "rounded-r-lg",
  "rounded-b-lg",
  "rounded-l-lg",
  "rounded-tl-lg",
  "rounded-tr-lg",
  "rounded-br-lg",
  "rounded-bl-lg",
  "rounded-t-md",
  "rounded-r-md",
  "rounded-b-md",
  "rounded-l-md",
  "rounded-tl-md",
  "rounded-tr-md",
  "rounded-br-md",
  "rounded-bl-md",
  "rounded-t-sm",
  "rounded-r-sm",
  "rounded-b-sm",
  "rounded-l-sm",
  "rounded-tl-sm",
  "rounded-tr-sm",
  "rounded-br-sm",
  "rounded-bl-sm",
];

```

**Comment by r4rmas** [comment]
I have the same issue.
If I change the `background-color`, the color does change but for some reason the `rounded-md` style get removed from the UI.

```
<Pressable onPress={onPress}>
    <View className="bg-red-800 py-1.5 px-3 rounded-md">
        <Text className="text-white text-sm">{text}</Text>
    </View>
</Pressable>
```

**Comment by flyfesan** [comment]
Not sure if this is related, but I'm experiencing a crash after a `className` change with this error:

```
› Opening the iOS simulator, this might take a moment.
iOS node_modules/expo-router/entry.js ░░░░░░░░░░░░░░░░  0.0% (0/1)
  nativewind getTransformOptions.dev true +21s
  nativewind getTransformOptions.platform ios +0ms
  nativewind getTransformOptions.virtualModulesPossible true +0ms
  nativewind virtualModules /Users/user/project/node_modules/react-native-css-interop/.cache/ios.js +1ms
  nativewind virtualModules.isDev true +0ms
  nativewind virtualModules.size 0 +0ms
  nativewind virtualModules.fastRefresh enabled +0ms
  nativewind getCSSForPlatform: ios +21s
  nativewind Start development Tailwind CLI +0ms
iOS node_modules/expo-router/entry.js ▓▓░░░░░░░░░░░░░░ 16.0% (156/390)
  nativewind resolveRequest.input /Users/project/global.css +103ms
  nativewind resolveRequest.resolvedTo: /Users/user/project/node_modules/react-native-css-interop/.cache/ios.js +0ms
  nativewind resolveRequest.development: true +0ms
iOS node_modules/expo-router/entry.js ▓▓▓▓▓▓▓▓▓▓▓░░░░░ 73.4% ( 897/1047)
  nativewind Finished initial development Tailwind CLI +374ms
  nativewind virtualStyles.initial ios +269ms
  nativewind Start stringify +20ms
iOS Bundled 618ms node_modules/expo-router/entry.js (1547 modules)
  nativewind resolveRequest.input /Users/user/project/global.css +1m
  nativewind resolveRequest.resolvedTo: /Users/user/project/node_modules/react-native-css-interop/.cache/ios.js +0ms
  nativewind resolveRequest.development: true +0ms
  nativewind resolveRequest.platform: ios +0ms
  nativewind Tailwind CLI detected new styles +1m
  nativewind virtualStyles.update ios +104ms
  nativewind Start stringify +3ms
  nativewind Output size: 2182 bytes +0ms
  nativewind virtualStyles.emit ios +0ms
/Users/user/project/node_modules/@expo/cli/build/src/utils/errors.js:130
    throw error;
    ^

TypeError: Cannot read properties of undefined (reading 'addedFiles')
    at DependencyGraph._onHasteChange (/Users/user/project/node_modules/metro/src/node-haste/DependencyGraph.js:99:18)
    at FileMap.<anonymous> (/Users/user/project/node_modules/metro/src/node-haste/DependencyGraph.js:68:14)
    at FileMap.emit (node:events:521:24)
    at Object.onChange (/Users/user/project/node_modules/react-native-css-interop/src/metro/index.ts:344:15)
    at ChildProcess.<anonymous> (/Users/user/project/node_modules/nativewind/src/metro/tailwind/v3/index.ts:70:23)
    at ChildProcess.emit (node:events:509:20)
    at emit (node:internal/child_process:956:14)
    at processTicksAndRejections (node:internal/process/task_queues:91:21)

Node.js v25.9.0
error Command failed.
Exit code: 7
Command: expo
Arguments: start
Directory: /Users/user/project/
Output:

info Visit https://yarnpkg.com/en/docs/cli/exec for documentation about this command.
```

This is with minimal manual installation of nativewind. (I'm new to this ecosystem, and I couldn't find anything in the internet related to this).

**Comment by wangfengming** [comment]
https://github.com/nativewind/nativewind/pull/1795/changes

I'v run ` DEBUG="nativewind,react-native-css-interop" npx expo start` ，output log:

```
 nativewind Tailwind CLI detected new styles +1m
    nativewind virtualStyles.update android +1m
    react-native-css-interop Features {"transformPercentagePolyfill":false,"disableGapPercentageValues":false,"disableDisplayBlock":false,"disableAlignContentEvenly":false,"disablePositionStatic":false} +0ms
    react-native-css-interop Grouping /^group(\/.*)?/ +0ms
    react-native-css-interop Start lightningcss +0ms
    react-native-css-interop Found 15386 rules to process +554ms
    react-native-css-interop Exiting lightningcss +391ms
    react-native-css-interop Found 15372 valid rules +56ms
    nativewind Start stringify +1s
    nativewind Output size: 1751219 bytes +50ms
    nativewind virtualStyles.emit android +0ms
    nativewind virtualStyles.emit failed TypeError: changes.modifiedFiles is not iterable
      at DependencyGraph._onHasteChange (/Users/wangfengming.18/Dev/box/home-box-client/node_modules/.pnpm/metro@0.83.6/node_modules/metro/src/node-haste/DependencyGraph.js:100:18)
      at FileMap.<anonymous> (/Users/wangfengming.18/Dev/box/home-box-client/node_modules/.pnpm/metro@0.83.6/node_modules/metro/src/node-haste/DependencyGraph.js:68:14)
      at FileMap.emit (node:events:536:35)
      at emitHasteFileChange (/Users/wangfengming.18/Dev/box/home-box-client/node_modules/.pnpm/react-native-css-interop@0.2.5_react-native-reanimated@4.2.1_react-native-worklets@0.7._0f6fb3118d0e83ab841aa35fe9d241b9/node_modules/react-native-css-interop/src/metro/index.ts:304:29)
      at Object.onChange (/Users/wangfengming.18/Dev/box/home-box-client/node_modules/.pnpm/react-native-css-interop@0.2.5_react-native-reanimated@4.2.1_react-native-worklets@0.7._0f6fb3118d0e83ab841aa35fe9d241b9/node_modules/react-native-css-interop/src/metro/index.ts:389:9)
      at ChildProcess.<anonymous> (/Users/wangfengming.18/Dev/box/home-box-client/node_modules/.pnpm/nativewind@4.2.5_react-native-reanimated@4.2.1_react-native-worklets@0.7.4_@babel+core@_027dde1408ee1ba5f7ea93bd0639ed05/node_modules/nativewind/src/metro/tailwind/v3/index.ts:70:23)
      at ChildProcess.emit (node:events:524:28)
      at emit (node:internal/child_process:950:14)
      at processTicksAndRejections (node:internal/process/task_queues:83:21)
```

`changes.changedFiles` is the property name of the old FileMap API; `changes.modifiedFiles` is the new one. `react-native-css-interop@0.2.5` did not keep up with this change, causing `changes.modifiedFiles` to be undefined, throwing a `TypeError: not iterable` when expanded, and CSS hot reloading silently fails in the final step.

After I added `modifiedFiles: new Set([[filePath, { isSymlink: false, modifiedTime }]]),`, the log no longer outputs errors, but hot reloading still doesn't work. Output is:
```
  nativewind Tailwind CLI detected new styles +20s
  nativewind virtualStyles.update android +19s
  react-native-css-interop Features {"transformPercentagePolyfill":false,"disableGapPercentageValues":false,"disableDisplayBlock":false,"disableAlignContentEvenly":false,"disablePositionStatic":false} +0ms
  react-native-css-interop Grouping /^group(\/.*)?/ +0ms
  react-native-css-interop Start lightningcss +0ms
  react-native-css-interop Found 15387 rules to process +358ms
  react-native-css-interop Exiting lightningcss +393ms
  react-native-css-interop Found 15373 valid rules +25ms
  nativewind Start stringify +795ms
  nativewind Output size: 1751288 bytes +49ms
  nativewind virtualStyles.emit android +0ms
```

**Comment by wangfengming** [comment]
ModifiedFiles requires relative paths, and everything worked fine after I modified the code in node_modules:

```
        const modifiedTime = Date.now();
        const rootDir = fileSystem?.lookup("")?.realPath;
        const relativePath = rootDir ? path_1.default.relative(rootDir, filePath) : filePath;  // <=== ✅
        haste.emit("change", {
            eventsQueue: [
                {
                    filePath,
                    metadata: {
                        modifiedTime: Date.now(),
                        size: 1,
                        type: "virtual",
                    },
                    type: "change",
                },
            ],
            get rootDir() {
                const rootDir = fileSystem?.lookup("");
                (0, assert_1.default)(rootDir?.exists && typeof rootDir.realPath === "string");
                return rootDir.realPath;
            },
            changes: {
                addedDirectories: EMPTY_SET,
                removedDirectories: EMPTY_SET,
                addedFiles: EMPTY_SET,
                removedFiles: EMPTY_SET,
                changedFiles: new Set([[filePath, { isSymlink: false, modifiedTime }]]),
                modifiedFiles: new Set([[relativePath, { isSymlink: false, modifiedTime }]]), // <=== ✅
            },
        });
```

@marklawlor

**Comment by wangfengming** [comment]
https://github.com/react/metro/blob/main/packages/metro-file-map/src/flow-types.js#L467

@marklawlor

**Comment by wangfengming** [comment]
0.84.2 use `eventsQueue`:  https://github.com/react/metro/blob/v0.84.2/packages/metro-file-map/types/flow-types.d.ts#L80 .
0.84.3 use `modifiedFiles`: https://github.com/react/metro/blob/v0.84.3/packages/metro-file-map/types/flow-types.d.ts#L84 .

The old version did not use `changedFiles`, so the `changedFiles` attribute can be removed.

**Comment by wangfengming** [comment]
@kitten

**Comment by wangfengming** [comment]
https://github.com/nativewind/nativewind/pull/1809

Just noticed that the issue has been resolved in this PR. When will this be released?

**Comment by is-it-ayush** [comment]
@onyedikachi23! You can close this issue. Nativewind release [4.2.6](https://github.com/nativewind/nativewind/releases/tag/nativewind%404.2.6) fixes this problem. I've verified the issue occurs on `4.2.5` and is fixed after upgrading to `4.2.6`.
